src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.product__review-box {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.product__stars {
+  display: flex;
+}
+
+.product__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the image path in `background-image` is correct and that the image exists at the specified location. If the image is missing, it will result in a broken star icon in the UI.